dist/jowl.spec

```diff
@@ -0,0 +1,90 @@
+Name:           jowl
+Version:        2.0.2
+Release:        1%{?dist}
+Summary:        CLI for JSON operations with Lodash
+
+%global gittag v%{version}
+
+License:        MIT
+URL:            https://jowl.app
+Source0:        https://github.com/daxelrod/%{name}/archive/%{gittag}/%{name}-%{version}.tar.gz
+# Also at       https://registry.npmjs.org/{name}/-/{name}-{version}.tgz
+# however, npmjs.org has a build product which does not contain docs or tests
+
+# The following sources are generated by running
+# nodejs-packaging-bundler against Source0:
+Source1:        %{name}-%{version}-nm-prod.tgz
+Source2:        %{name}-%{version}-nm-dev.tgz
+Source3:        %{name}-%{version}-bundled-licenses.txt
+
+BuildArch:      noarch
+ExclusiveArch:  %{nodejs_arches} noarch
+Requires:       nodejs
+BuildRequires:  nodejs-devel
+BuildRequires:  yarnpkg
+
+%description
+Jowl is a command-line filter for JSON expressions that uses plain JavaScript
+with Lodash. It takes JSON on standard in, and writes pretty-printed JSON to
+standard out.
+
+%prep
+%setup -q -n %{name}-%{version}
+cp %{SOURCE3} .
+# Setup bundled runtime(prod) node modules
+tar xfz %{SOURCE1}
+mkdir -p node_modules
+pushd node_modules
+ln -s ../node_modules_prod/* .
+ln -s ../node_modules_prod/.bin .
+popd
+
+%build
+#nothing to do
+
+%install
+mkdir -p %{buildroot}%{nodejs_sitelib}/%{name}
+cp -pr package.json src/bin/jowl.js src/lib/ \
+    %{buildroot}%{nodejs_sitelib}/%{name}
+# Copy over bundled nodejs modules
+cp -pr node_modules node_modules_prod \
+    %{buildroot}%{nodejs_sitelib}/%{name}
+
+mkdir -p %{buildroot}%{nodejs_sitelib}/%{name}/bin
+install -p -D -m0755 src/bin/jowl.js %{buildroot}%{nodejs_sitelib}/%{name}/bin/jowl
```

Comment:
Why not make this a symlink?

```bash
ln -sr %{buildroot}%{nodejs_sitelib}/%{name}/jowl.js %{buildroot}%{nodejs_sitelib}/%{name}/bin/jowl
```

Reply:
So it turns out when I try this, it messes up Node's library loading logic:

```
[root@e78b4a886621 /]# ls -al /usr/bin/jowl
lrwxrwxrwx. 1 root root 35 Sep 16 02:06 /usr/bin/jowl -> /usr/lib/node_modules/jowl/bin/jowl
[root@e78b4a886621 /]# ls -al /usr/lib/node_modules/jowl/bin/jowl 
lrwxrwxrwx. 1 root root 10 Sep 16 02:06 /usr/lib/node_modules/jowl/bin/jowl -> ../jowl.js
[root@e78b4a886621 /]# /usr/bin/jowl
...
Error: Cannot find module '../lib/format'
Require stack:
- /usr/lib/node_modules/jowl/jowl.js
```

That must be why the tape specfile I got this from did it without the symlink

Reply:
That's weird, but okay...